src/server.ts

```diff
@@ -394,7 +395,7 @@ const schemaRequestHandlerWrapper = (connection: IConnection, uri: string): Prom
 
 const schemaRequestService = schemaRequestHandlerWrapper.bind(this, connection);
 
-export const customLanguageService = getCustomLanguageService(schemaRequestService, workspaceContext);
+let customLanguageService: LanguageService;
```

Comment:
I think reassigning inside of onInitiailize seems to work. That's what vscode-json-languageserver does: https://github.com/vscode-langservers/vscode-json-languageserver/blob/master/src/jsonServer.ts#L80. I think we'd probably still have to do:
```suggestion
let customLanguageService = getCustomLanguageService(schemaRequestService, workspaceContext, ClientCapabilities.LATEST);
```
just in case though